data/obs-packaging/cc-oci-runtime/README.md

```diff
@@ -8,10 +8,14 @@ With these files we generated Fedora and Ubuntu packages for this component.
 
 ``./update_runtime.sh [VERSION]``
 
+The ``VERSION`` parameter is optional, it can be a tag, a branch or a git hash.
+
+If VERSION is not specified, it will be determined automatically from the
```

Comment:
If the ``VERSION`` parameter is not specified, the top-level ``configure.ac`` file will determine its value automatically.